app/Console/Commands/Environment/DatabaseSettingsCommand.php

```diff
@@ -98,7 +98,7 @@ public function handle(): int
         } elseif ($this->variables['DB_CONNECTION'] === 'sqlite') {
             $this->variables['DB_DATABASE'] = $this->option('database') ?? $this->ask(
                 'Database Path',
-                config('database.connections.sqlite.database', database_path('database.sqlite'))
+                config('database.connections.sqlite.database', 'database.sqlite')
```

Comment:
still has database_path() in config/database.php

Reply:
> still has database_path() in config/database.php

Yes, that's why it needs to be removed here.